pkg/pillar/cmd/volumemgr/updatestatus.go

```diff
@@ -138,6 +145,10 @@ func doUpdateContentTree(ctx *volumemgrContext, status *types.ContentTreeStatus)
 			}
 			totalSize += blob.TotalSize
 			currentSize += blob.CurrentSize
+			if blob.IsLoaded {
+				blob.State = types.VERIFIED
+			}
```

Comment:
This would be easier if we could have `blob.State = types.LOADED`

Reply:
@adarsh-zededa when adding a new state there is a String and a ZSwState function that you need to update in types/types.go

Reply:
Added a LOADED state.